### ISSUE TITLE
Exit watcher is producing unexpected errors

### ISSUE THREAD

**Issue by shanefontaine** [hop-protocol/hop#291]
The exit watcher is producing errors that I would not expect it to be producing at this time. These can be observed in the logs.

### Error 1 -- `messagePairs` error

This error has only recently started happening and occurs on the DAI bonder only (I believe). I believe it might have to do with the updated `eth-optimism/contracts` package, but that is a loose guess.

```
watcher: xDomainMessageRelay, label: optimism.DAI, host: 935fdc59ea86 poll check error: Cannot destructure property 'message' of 'messagePairs[0]' as it is undefined.
```

### Error 2 -- `unable to find tx with hash`

The Optimism watcher (on all bonders) produces this type of error very regularly. I would not expect it to be attempting transactions like this. My loose guess is that it is not waiting 7 days for the transaction to be able to be exited(or the transaction is not exited exactly at 7 days but maybe some time a little later).

```
watcher: xDomainMessageRelay, label: optimism.ETH, host: ec23ff115048poll check error: unable to find tx with hash: 0x79f2af4515d7e4552d00f34cffa9debf909bb0ee5179ea9fe4104f39680577dd
```

**Comment by shanefontaine** [comment]
Error 1 handled with 839da108109dba3b57a56c376079efbc9d52877b

**Comment by shanefontaine** [comment]
Closed with #293
